db/comdb2.c

```diff
@@ -621,6 +621,7 @@ int gbl_check_wrong_db = 1;
 int gbl_broken_max_rec_sz = 0;
 int gbl_private_blkseq = 1;
 int gbl_use_blkseq = 1;
+int gbl_reorder_blkseq_no_deadlock = 1;
```

Comment:
you're reordering the socksql stream, not blkseq - maybe change this to 'gbl_reorder_socksql_no_deadlock'?